test_quality_of_service/include/test_quality_of_service/qos_utilities.hpp

```diff
@@ -83,8 +83,15 @@ class BaseQosRclcppTestFixture : public ::testing::Test
 
 class QosRclcppTestFixture : public BaseQosRclcppTestFixture
 {
+public:
+  QosRclcppTestFixture()
+  {
+    rmw_implementation_str = std::string(rmw_get_implementation_identifier());
+  }
+
 protected:
   const std::string this_rmw_implementation{RCUTILS_STRINGIFY(RMW_IMPLEMENTATION)};
+  std::string rmw_implementation_str;
```

Comment:
Sorry, I'm just noticing this.  What's the difference between `this_rmw_implementation` and `rmw_implementation_str`?

Reply:
Fixed https://github.com/ros2/system_tests/pull/551/commits/4d873ee34c5869f13d97b5b08b63adff23066ce7